### PR TITLE
Import/Export Feature

### PR DESCRIPTION
**Pull request by kdvalin**
This PR allows for scenes to be exported to a user's device and then imported at a later date.

### CODE REVIEW

**Comment by Jlu18** [comment]
Can you update the backend [import/export PR](https://github.com/engaging-computing/MYR-backend/pull/160) so I can test it on my machine.

File: src/components/structural/header/ProjectView.js [hunk 1]
```diff
@@ -423,6 +435,27 @@ class Project extends React.Component {
                                     }
                                     <span>&nbsp;</span>Preview
                                 </Button>
+                                <input
+                                    type="file"
+                                    accept="application/json"
+                                    ref={this.uploadRef}
+                                    onChange={(e) => this.props.importFunc(this.props.user.uid, e.target.files)}
```
Comment (Jlu18) [inline src/components/structural/header/ProjectView.js:442]: Also returns error when user is not logged in.

File: src/components/structural/header/ProjectView.js [hunk 2]
```diff
@@ -351,6 +356,13 @@ class Project extends React.Component {
                 </ListItemIcon>
                 <ListItemText inset primary="Send" />
             </MenuItem>
+            <MenuItem
+                onClick={() => this.props.exportFunc(this.props.user.uid, this.state.projectId)}>
+                <ListItemIcon>
+                    <Icon className="material-icons">download</Icon>
+                </ListItemIcon>
+                <ListItemText inset primary="Export Scene" />
+            </MenuItem>
```
Comment (Jlu18) [inline src/components/structural/header/ProjectView.js:365]: Example project also has an option to export the scene but unable to.

File: src/components/structural/header/ProjectView.js (hunk 1 above)
Comment (Jlu18) [reply src/components/structural/header/ProjectView.js:442]: Probably hide the option if user is not logged in (or import the scene without saving as a scene)

File: src/components/structural/header/ProjectView.js [hunk 3]
```diff
@@ -423,6 +435,27 @@ class Project extends React.Component {
                                     }
                                     <span>&nbsp;</span>Preview
                                 </Button>
+                                <input
```
Comment (Jlu18) [inline src/components/structural/header/ProjectView.js:438]: Is there any reason why this input tag is outside of the import Button? It seems to be working fine but I think it should be a child of the Button.

**Review by Jlu18 — CHANGES_REQUESTED** [review]
The import and export options should be disabled (or hide away) if the user is not logged in, as well as if the user is on example project tab (since it doesn't seem like you can export those)

File: src/components/structural/header/ProjectView.js (hunk 2 above)
Comment (Jlu18) [reply src/components/structural/header/ProjectView.js:366]: Exporting example scene still gave 404 error. Is this intentional or do we want to give user option to export the example scene?

**Review by Jlu18 — CHANGES_REQUESTED** [review]

**Comment by kdvalin** [comment]
Bumping to next version since I needed to update AFrame

**Review by Jlu18 — DISMISSED** [review]

**Pull request closed without merging**